### PR TITLE
chore(cli): return with an exit code of 1 on supplying an unknown command

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Docs
- [ ] Underlying tools
- [x] Other, please describe: Supplying an unknown command returns an exit code of 1.

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**

N/A

### CODE REVIEW

File: packages/@vue/cli/__tests__/__e2e__/args.spec.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+const runAsync = async (args, options) => execa(CLI_PATH, args, options)
+
+test('suggests matching command', async () => {
+  const { code, stdout } = await runAsync(['confgi'], { reject: false })
+
+  // Assertions
+  expect(code).toBe(1)
+  expect(stdout).toContain('Did you mean config?')
+})
```
Comment (fangbinwei) [inline packages/@vue/cli/__tests__/__e2e__/args.spec.js:14]: Why create a new file?

Comment (jamesgeorge007) [reply packages/@vue/cli/__tests__/__e2e__/args.spec.js:14]: It is always better to have an abstraction layer between `e2e` and unit tests.

Comment (fangbinwei) [reply packages/@vue/cli/__tests__/__e2e__/args.spec.js:14]: I agree. If so, it's better to refactor all project tests folder. Only create a `__e2e` for this `args.spec.js` will make directory structure more complicated. I think consistency first.

Comment (jamesgeorge007) [reply packages/@vue/cli/__tests__/__e2e__/args.spec.js:14]: In that case, let's keep the directory structure as it is for now.

**Review by fangbinwei — APPROVED** [review]

**Pull request merged**